docs/book/src/tasks/automated-machine-management/machine_deletions.md

```diff
@@ -0,0 +1,127 @@
+  * Pods belonging to an existing DaemonSet (orphaned DaemonSet Pods have to be evicted as well)
+  * Mirror Pods, i.e. Pods with the `kubernetes.io/config.mirror` annotation (usually static Pods managed by kubelet, like `kube-apiserver`)
+* If there are no (more) Pods that have to be evicted and all Pods that have been evicted are gone, Node drain is completed
+* Otherwise an eviction will be triggered for all Pods that have to be evicted. There are various reasons why an eviction call could fail:
+  * The eviction would violate a PodDisruptionBudget, i.e. not enough Pod replicas would be available if the Pod would be evicted
+  * The namespace is in terminating, in this case the `kube-controller-manager` is responsible for setting the `.metadata.deletionTimestamp` on the Pod
+  * Other errors, e.g. a connection issue when calling the eviction API at the workload cluster
+* Please note that when an eviction goes through, this only means that the `.metadata.deletionTimestamp` is set on the Pod, but the 
+  Pod also has to be terminated and the Pod object has to go away for the drain to complete.
+* These steps are repeated every 20s until all relevant Pods have been drained from the Node
+
+Special cases:
+* If the Node doesn't exist anymore, Node drain is entirely skipped
+* If the Node is `unreachable` (i.e. the Node `Ready` condition is in status `Unknown`):
+  * Pods with `.metadata.deletionTimestamp` more than 1s in the past are ignored
+  * Pod evictions will use 1s `GracePeriodSeconds`, i.e. the `terminationGracePeriodSeconds` field from the Pod spec will be ignored.
+  * Note: PodDisruptionBudgets are still respected, because both of these changes are only relevant if the call to trigger the Pod eviction goes through.
+    But Pod eviction calls are rejected when PodDisruptionBudgets would be violated by the eviction.
+
+### Observability
+
+The drain process can be observed through various logs and the `DrainingSucceeded` condition on the Machine.
+
+**Example logs**
+
+When cordoning the Node:
+```text
+I0830 12:50:13.961156      17 machine_controller.go:716] "Cordoning Node" ... Node="my-cluster-md-0-wxtcg-mtg57-k9qvz"
+```
+
+When starting the drain:
+```text
+I0830 12:50:13.961156      17 machine_controller.go:716] "Draining Node" ... Node="my-cluster-md-0-wxtcg-mtg57-k9qvz"
+```
+
+Immediately before Pods are evicted:
+```text
+I0830 12:52:58.739093      17 drain.go:172] "Drain not completed yet, there are still Pods on the Node that have to be drained" ... Node="my-cluster-md-0-wxtcg-mtg57-ssfg8" podsToTriggerEviction="test-namespace/nginx-deployment-6886c85ff7-4r297, test-namespace/nginx-deployment-6886c85ff7-5gl2h, test-namespace/nginx-deployment-6886c85ff7-64tf9, test-namespace/nginx-deployment-6886c85ff7-9k5gp, test-namespace/nginx-deployment-6886c85ff7-9mdjw, ... (5 more)" podsWithDeletionTimestamp="kube-system/calico-kube-controllers-7dc5458bc6-rdjj4, kube-system/coredns-7db6d8ff4d-9cbhn"
```

Comment:
What about
```suggestion
I0830 12:52:58.739093      17 drain.go:172] "Drain starting" ... Node="my-cluster-md-0-wxtcg-mtg57-ssfg8" podsToTriggerEviction="test-namespace/nginx-deployment-6886c85ff7-4r297, test-namespace/nginx-deployment-6886c85ff7-5gl2h, test-namespace/nginx-deployment-6886c85ff7-64tf9, test-namespace/nginx-deployment-6886c85ff7-9k5gp, test-namespace/nginx-deployment-6886c85ff7-9mdjw, ... (5 more)" podsWithDeletionTimestamp="kube-system/calico-kube-controllers-7dc5458bc6-rdjj4, kube-system/coredns-7db6d8ff4d-9cbhn"
```

Reply:
Not sure. The drain is not actually starting here. This log line is usually logged many times (one in each reconcile)

Reply:
Even if I write "Start evicting Pods" it's not always true, because we are only evicting if there are still "pods to evict"

Reply:
I think in general I think it's a good idea to tell folks very very clearly why the drain is not completed